### PR TITLE
Nerfs jelly people slightly and makes them roundstart available

### PR DESCRIPTION
**Pull request by Moltijoe**
More round start races that aren't lizards is always good.
I removed the 0.5 burn mod so they aren't busted (slime, luminescent and stargazers still get it)
The only fancy things they have are the limb loss on low blood / limb regrow ability
and the inverted toxin damage

:cl:  
rscadd: Jelly people can be played at round start
/:cl:

### CODE REVIEW

**Comment by cowbot92** [comment]
Solid PR, think more round start races are always good and jelly people are unique enough (rip skrem have fun with the lore)

**Comment by Burger1998** [comment]
xenobio nerf in a trenchcoat. good.

**Comment by ThatLing** [comment]
This conflicts with the preferences rework, so this PR will be on-hold until then. You will have add additional things if you want them to be available roundstart

**Comment by Moltijoe** [comment]
> This conflicts with the preferences rework, so this PR will be on-hold until then. You will have add additional things if you want them to be available roundstart

assumed it would happen 👍

**Comment by Marmio64** [comment]
i feel like they're still an upgrade to being a human, where the other races have upsides but yet are clearly designed as a sidegrade/downgrade. In addition I feel like they just dont need to be another race, we have a bunch already, why not just keep slimepeople to xenobio.

**Comment by Moltijoe** [comment]
> i feel like they're still an upgrade to being a human, where the other races have upsides but yet are clearly designed as a sidegrade/downgrade. In addition I feel like they just dont need to be another race, we have a bunch already, why not just keep slimepeople to xenobio.

these aren't slimepeople, they're jelly people they don't have the body split thing and they most certainly are not a direct upgrade to humans. being put into a cryotube kills them faster than it heals, charcoal kills them, they die ridiculously quickly if they spend even slightly too long in a spaced room.

**Comment by Marmio64** [comment]
> > i feel like they're still an upgrade to being a human, where the other races have upsides but yet are clearly designed as a sidegrade/downgrade. In addition I feel like they just dont need to be another race, we have a bunch already, why not just keep slimepeople to xenobio.
> 
> these aren't slimepeople, they're jelly people they don't have the body split thing and they most certainly are not a direct upgrade to humans. being put into a cryotube kills them faster than it heals, charcoal kills them, they die ridiculously quickly if they spend even slightly too long in a spaced room.

I know precisely what they are. They also barely worry about limb loss, don't worry about many conventional toxins, and have very high heat resistance.

**Comment by Moltijoe** [comment]
> I know precisely what they are. They also barely worry about limb loss, don't worry about many conventional toxins, and have very high heat resistance.

IPC, plasmeme, and preternis don't really need to worry about limb loss either (probably poly too, not sure how much their wound armour factors in)
toxins are neither all that prevalent nor are they hard to counter, all it takes is one charcoal pill to counter most poisons
not to mention, if you're trying to kill a jellyperson, just used charcoal as your poison of choice.
also, heat mod of 0.5 isn't nearly that powerful and it's more than outweighed by the cold mod of 6,

**Comment by Marmio64** [comment]
> > I know precisely what they are. They also barely worry about limb loss, don't worry about many conventional toxins, and have very high heat resistance.
> 
> IPC, plasmeme, and preternis don't really need to worry about limb loss either (probably poly too, not sure how much their wound armour factors in) toxins are neither all that prevalent nor are they hard to counter, all it takes is one charcoal pill to counter most poisons not to mention, if you're trying to kill a jellyperson, just used charcoal as your poison of choice. also, heat mod of 0.5 isn't nearly that powerful and it's more than outweighed by the cold mod of 6,

even if they're balanced i still see no point in adding more races, especially if they're supposed to be xenobio exclusive.  Next we'll be adding super weak golems that can wear clothes as roundstart species, which like 1 person will play and confuse everyone because they'll all think it's a servant golem.

**Comment by Moltijoe** [comment]
> even if they're balanced i still see no point in adding more races, especially if they're supposed to be xenobio exclusive. Next we'll be adding super weak golems that can wear clothes as roundstart species, which like 1 person will play and confuse everyone because they'll all think it's a servant golem.

Theres no reason not to add more species if they fit, even if only one person plays them, that's still one person playing the species they enjoy. 
Jellypeople aren't xenobio exclusive, they just don't exist, all the upgraded jellyperson species will still be exclusively xenobio.
Also, likening this to golems isn't that reasonable as they have so many differences from other species that they shouldn't ever be considered for proper crewmates. That would be like me saying shadowpeople should be round start.

**Review by ThatLing — CHANGES_REQUESTED** [review]
This will need changes to work with the preferences. Message me on discord if you need help

**Comment by Moltijoe** [comment]
> This will need changes to work with the preferences. Message me on discord if you need help

ah, didn't notice prefs were full merged, assumed they were just test merged still

File: code/modules/mob/living/carbon/human/species_types/jellypeople.dm [hunk 1]
```diff
@@ -68,17 +68,52 @@
 	qdel(consumed_limb)
 	H.blood_volume += 20
 
+/datum/species/jelly/random_name(gender,unique,lastname)//they have no lore, just use human names for now i guess
+	if(unique)
+		return random_unique_name()
+	return random_unique_name()
+
+/datum/species/jelly/get_features()//they currently have no special features
```
Comment (ThatLing) [inline code/modules/mob/living/carbon/human/species_types/jellypeople.dm]: Remove this if they don't

File: code/modules/mob/living/carbon/human/species_types/jellypeople.dm [hunk 2]
```diff
@@ -68,17 +68,52 @@
...
+		SPECIES_PERK_ICON = "tint",
+		SPECIES_PERK_NAME = "Regenerate Limbs",
+		SPECIES_PERK_DESC = "Being made entirely of jelly means [plural_form] can reform lost limbs from nothing assuming they have enough extra to spare.",
+	))
+
+	return to_add
+
 // Slimes have both NOBLOOD and an exotic bloodtype set, so they need to be handled uniquely here.
-// They may not be roundstart but in the unlikely event they become one might as well not leave a glaring issue open.
+// They may not be roundstart but in the unlikely event they become one might as well not leave a glaring issue open. <- yeah, i'm trying to add them as roundstart
```
Comment (ThatLing) [inline code/modules/mob/living/carbon/human/species_types/jellypeople.dm]: Remove this useless comment

**Review by ThatLing — CHANGES_REQUESTED** [review]

**Comment by ThatLing** [comment]
@Moltijoe Is this still a draft?

**Comment by Moltijoe** [comment]
> @Moltijoe Is this still a draft?

Yes until someone tells me how to change the species icon, since it still just shows an ethereal rather than jellyperson

**Review by ThatLing — APPROVED** [review]

**Comment by JamieD1** [comment]
> > @Moltijoe Is this still a draft?
> 
> Yes until someone tells me how to change the species icon, since it still just shows an ethereal rather than jellyperson

this done?

**Comment by Moltijoe** [comment]
> > > @Moltijoe Is this still a draft?
> > 
> > 
> > Yes until someone tells me how to change the species icon, since it still just shows an ethereal rather than jellyperson
> 
> this done?

Yeah

**Comment by JamieD1** [comment]
Lets see how we go shall we

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**